packages/nginx-amplify-agent/rpm/nginx-amplify-agent.spec

```diff
@@ -23,13 +23,22 @@ BuildRequires: python3-pip
 %if 0%{?amzn} >= 2
 Requires: python3 >= 3.7
 Requires: python3-requests
-%else
```

Comment:
Looks like the `else` block has been removed and 2 new checks for rhel8 and rhel9 have been added. What about use cases which were previously handled in the `else` block but are not `rhel8` or `rhel9` (i.e., none of the Python Requires logic will take effect for those use caseS).

Reply:
We currently list RHEL 8 and Amazon Linux as supported rpm-based distros only, so no reason to worry about other ones (e.g. opensuse/SLES or Fedora).

In theory current spec should work flawlessly on RHEL derivatives such as AlmaLinux and RockyLinux, but as these are binary compatible with RHEL, we don't need to worry either.